docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:13
+    env_file:
+      # Use template env file ./docker/secrets.env.sample
+      #   and generate your container secrets
+      - ./docker/secrets.env
+    environment:
+      POSTGRES_DB: anemperfeina
+      POSTGRES_USER: anemperfeina
+
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build:
+      context: .
+    image: anemperfeina:1.0
+    depends_on:
+      - db
+    ports:
+      - 8000:8000
+
```

Comment:
Current code should be mounted into the container to simplify and enhance development workflow.

It can be achieved with something like:
```
    volumes:
      - .:/anem-per-feina/
```

With this, the base image is not needed to be recreated for each change. Also Djangos reload-on-changes default behavior will be preserved.

Reply:
Would be great to the app volume mounted as @XaviTorello suggest,  so we can development already in a docker environment.